### PR TITLE
[LUM-282] Add pagination to all tables

### PR DESCRIPTION
**Pull request by ThibaultJRD**
- Add paginations
- Match with new chain bridge endpoints

### CODE REVIEW

File: src/redux/models/models/blocks.ts [hunk 1]
```diff
@@ -66,14 +68,16 @@ const blocks = createModel<RootModel>()({
...
-                const block = await ApiBlocks.getBlock(id);
+                const [block] = await Api.getBlock(id);
 
                 dispatch.blocks.SET_BLOCK(block);
             } catch (e) {}
         },
 
         addBlock(block: BlocksModel) {
-            dispatch.blocks.ADD_NEW_BLOCK(block);
+            console.log(block);
```
Comment (Segfaultd) [inline src/redux/models/models/blocks.ts:78]: Oopsi

File: src/redux/models/models/transactions.ts [hunk 2]
```diff
@@ -65,14 +67,16 @@ const transactions = createModel<RootModel>()({
...
-                const transaction = await ApiTransactions.getTransaction(id);
+                const [transaction] = await Api.getTransaction(id);
 
                 dispatch.transactions.SET_TRANSACTION(transaction);
             } catch (e) {}
         },
 
         addTransaction(transaction: TransactionsModel) {
-            dispatch.transactions.ADD_NEW_TRANSACTION(transaction);
+            console.log(transaction);
```
Comment (Segfaultd) [inline src/redux/models/models/transactions.ts:77]: Oopsi

**Review by Segfaultd — CHANGES_REQUESTED** [review]

**Review by Segfaultd — CHANGES_REQUESTED** [review]
There are a few leftover that requires changes.

Also, dark mode isn't working as expected on pagination components.

**Comment by ThibaultJRD** [comment]
> Also, dark mode isn't working as expected on pagination components.

Dark mode on pagination components is fixed 😄

**Pull request merged**